xbitinfo/bitround.py

```diff
@@ -37,18 +37,25 @@ def _keepbits_interface(da, keepbits):
         else:
             raise ValueError(f"name {v} not for in keepbits: {keepbits.keys()}")
     elif isinstance(keepbits, xr.Dataset):
+        assert keepbits.coords["inflevel"].shape <= (
+            1,
+        ), "length of 'threshold' dimension of keepbits need to be one"
```

Comment:
zero or one? is a non-dimensional coord also allowed? I think so

Reply:
Yes, both are consciously allowed.

Reply:
I would also rename threshold to inflevel.

Reply:
`dimension ... need to be one` sounds like it needs to be a dimension but non-dim coord also works